proposals/incubation/cert-manager.md

```diff
@@ -0,0 +1,138 @@
+# cert-manager Incubation Proposal
+
+cert-manager joined the CNCF as a sandbox project just over a year ago, and we firmly believe that the project is ready to take the next step and graduate to the Incubation stage.
+
+cert-manager provides reliable, automated Kubernetes-native management of X.509 certificates. It reduces the risk of certificate-related outages and enables PKI at scale, and is treated by many teams as de-facto part of the core components of a Kubernetes or OpenShift deployment.
+
+The project comprises cert-manager itself - which adds Kubernetes Custom Resource Definitions (CRDs) for certificate-related concepts, and provides controllers to manage them - along with a selection of subprojects which focus on solving other issues relating to certificate management.
+
+The main cert-manager repo has 8.4k stars on GitHub and hundreds of projects downstream which depend on cert-manager code in some way. Thousands of people have contributed to the project in some way and cert-manager's use and prominence remains strong.
+
+## Incubation Criteria Checklist
+
+Based on the [graduation criteria][GraduationCriteria], cert-manager seems to clearly tick the boxes required for incubation:
+
+### Production Use
+
+> Document that it is being used successfully in production by at least three independent end users
+
+cert-manager is used in production by _many_ more than three independent end users. Several major cloud native projects including [Istio][IstioCertManager], the [Kubernetes Cluster API][ClusterAPICertManager] and [Knative][KnativeCertManager] either suggest or require that cert-manager be installed.
+
+Walmart uses cert-manager too, and mentioned it in a [Keynote speech they gave][WalmartKeynote].
+
+According to the [companies-table DevStats dashboard][CompaniesTable] over a thousand different companies (plus ~1330 "Independents") have made some kind of contribution (defined as somebody who made a review, comment, commit, created PR or issue) to cert-manager. That implies that the floor of cert-manager's reach can be measured in the thousands, but there could be many more users who happily use cert-manager without contributing back or raising GitHub issues.
+
+One prominent example of a cert-manager user might be the CNCF themselves, using cert-manager [in the DevStats project][DevStatsCertManager]!
+
+In addition we might measure some production use through the many large companies who have seen fit to create addons for cert-manager to help it better integrate into their environments. That includes (at the time of writing):
+
+- at least [49 public repositories][ExternalDNSSolvers] implementing an external ACME DNS-01 solver
+- at least [8 public repositories][ExternalIssuers] implementing complete external issuers for certificates
+
+Finally, we track a few users in a [USERS.md file][USERSmd] on our repository, although that file doesn't fully represent the scale of cert-manager's prominence in clusters today.
+
```

Comment:
optional: We could list a couple of "production users" listed on the ArtifactHub page:
![image](https://user-images.githubusercontent.com/978965/154064685-c0a6aa3b-2a36-4915-a4c2-4f0002bf84c0.png)
https://artifacthub.io/packages/helm/cert-manager/cert-manager

Reply:
I like the idea and I'd not thought to look on ArtifactHub. That said, I think I'd prefer to not add these to the doc; we have plenty of orgs listed currently (and we only needed 3).